### ISSUE TITLE
Re-upload same version for airgap install

### ISSUE THREAD

**Issue by MikaelSmith** [replicatedhq/kots#761]
It should be possible to re-upload the same version of an airgap package in case something went wrong with the previous upload (for example, registry had to be rebuilt).

Currently when I try, it errors with
```
Version a477879 (137) cannot be installed because version a477879 (137) is newer
```

**Comment by marccampbell** [comment]
@MikaelSmith agreed. The previous (failed) upload should not even show up in the release list.
